### PR TITLE
Add 0001-Add-an-option-to-never-resolve-parent-path-when-vhd-.patch

### PR DESCRIPTION
**Pull request by Wescoeur**

### CODE REVIEW

**Comment by Wescoeur** [comment]
Koji build (XCP-ng 8.3, incoming, scratch): http://koji.xcp-ng.org/taskinfo?taskID=33834

File: SPECS/blktap.spec [hunk 1]
```diff
@@ -25,6 +25,8 @@ Conflicts: sm < 3.0.1
 
 Provides: blktap(nbd) = 2.0
 
+Patch1001: 0001-Add-an-option-to-never-resolve-parent-path-when-vhd-.patch
+
```
Comment (stormi) [inline SPECS/blktap.spec:29]:
```suggestion
# XCP-ng patches
# Required by XOSTOR
Patch1000: 0001-Add-an-option-to-never-resolve-parent-path-when-vhd-.patch

```

File: SPECS/blktap.spec [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Summary: blktap user space utilities
 Name: blktap
 Version: 3.53.0
-Release: 1%{?xsrel}%{?dist}
+Release: 1%{?xsrel}1%{?dist}
```
Comment (stormi) [inline SPECS/blktap.spec:6]:
```suggestion
Release: 1.1%{?xsrel}%{?dist}
```

**Review by stormi — APPROVED** [review]

**Comment by stormi** [comment]
Koji build (v8.3-incoming): http://koji.xcp-ng.org/taskinfo?taskID=33905

**Pull request merged**